be/src/vec/functions/comparison_equal_for_null.cpp

```diff
@@ -118,14 +115,47 @@ class FunctionEqForNull : public IFunction {
 
             auto* __restrict res = res_map.data();
             auto* __restrict l = null_map.data();
-            for (int i = 0; i < input_rows_count; ++i) {
-                res[i] &= (l[i] != 1);
-            }
+            _exec_nullable_inequal(res, l, input_rows_count, left_const);
 
             block.get_by_position(result).column = res_nullable_column->get_nested_column_ptr();
         }
         return Status::OK();
     }
+
+private:
+    static void _exec_nullable_equal(unsigned char* result, const unsigned char* left,
+                                     const unsigned char* right, size_t rows, bool left_const,
+                                     bool right_const) {
+        if (left_const && right_const) {
```

Comment:
recheck the function whether can SIMD

Reply:
checked for Release version. That's ok for auto-simd